### PR TITLE
[TS] LPS-156368 Show alert message when adding a documents and media field

### PR DESCRIPTION
**Pull request by evibudai**
Hi Team,

**Issue**
The users don't know when they add an upload field to a DDL form that the uploaded document does not inherit the DDL's permission.

Previous PR: https://github.com/liferay-forms/liferay-portal/pull/1904
Total diff after SF: https://github.com/brianchandotcom/liferay-portal/compare/9c09777398...d8af0e2e37

**Goal of this PR**
This fix needs to guarantee [LPS-161261](https://issues.liferay.com/browse/LPS-161261) will not happen.

**Proof**
https://user-images.githubusercontent.com/97447507/190367777-dfbca380-2bc9-41b4-a2ac-334bacf661bc.mp4

Please review my PR!

Thank you and best regards,
Évi

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/form_builder.jspf [hunk 1]
```diff
@@ -15,14 +15,22 @@
...
 if (Validator.isNotNull(script)) {
 	try {
 		DDMForm ddmForm = DDMUtil.getDDMForm(script);
 
+		for (DDMFormField ddmFormField : ddmForm.getDDMFormFields()) {
+			String dataType = ddmFormField.getType();
+
+			if (dataType.equals("ddm-documentlibrary")) {
+				hasDDMDocumentLibraryDataType = true;
+			}
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/form_builder.jspf:31]: Hi @evibudai, could you please change to: 

```
if (StringUtil.equals(ddmFormField.getType(), "ddm-documentlibrary")) {
	hasDDMDocumentLibraryDataType = true;

	break;
}
```

Thanks!

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Comment by carolmariaabb** [comment]
@alinedoleron could you please review this PR? Thanks!

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/form_builder.jspf (hunk 1 above)
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/form_builder.jspf:31]: Hi @evibudai, could you please change to: 

```
if (StringUtil.equals(ddmFormField.getType(), "ddm-documentlibrary")) {
	hasDDMDocumentLibraryDataType = true;

	break;
}
```

Thanks!

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/form_builder.jspf (hunk 1 above)
Comment (evibudai) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/form_builder.jspf]: @carolmariaabb  I changed it :)

**Review by carolmariaabb — APPROVED** [review]
LGTM!

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Review by alinedoleron — APPROVED** [review]
LGTM!

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**